tests/basic/volume_ops_test.go

```diff
@@ -1484,3 +1488,157 @@ var _ = Describe("{ResizeVolumeAfterFull}", func() {
+			log.FailOnError(err, "Unable to get the storage nodes")
+
+			// Select random Storage node for the test
+			if len(pxNodes) > 0 {
+				pxNode = GetRandomNode(pxNodes)
+			} else {
+				log.FailOnError(errors.New("No Storage Node Availiable"), "Error occured while selecting StorageNode")
+			}
+
+			log.Infof("The Selected node for Fast path label is %v : ", pxNode.Name)
+
+			// Remove if node-type label is set before the test
+			Inst().AppList = applist
+			RemoveLabelsAllNodes(k8s.NodeType, true, false)
+		})
+
+		stepLog = "Step 2: Schedule application and Add label on the selected storage node"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			var err error
+
+			// Add label on the selected node
+			err = Inst().S.AddLabelOnNode(pxNode, k8s.NodeType, k8s.FastpathNodeType)
+			log.FailOnError(err, fmt.Sprintf("Failed add label on node %s", pxNode.Name))
+			Inst().AppList = []string{"fio-fastpath-repl1"}
+			contexts = make([]*scheduler.Context, 0)
+			for i := 0; i < Inst().GlobalScaleFactor; i++ {
+				contexts = append(contexts, ScheduleApplications(fmt.Sprintf("fastpath-%d", i))...)
+			}
+			ValidateApplications(contexts)
+		})
+
+		stepLog = " Step 3: Get app volumes and Check fast path is active on the node"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			var err error
+			for _, ctx := range contexts {
+				var appVolumes []*volume.Volume
+				stepLog = fmt.Sprintf("get volumes for %s app", ctx.App.Key)
+				Step(stepLog, func() {
+					log.InfoD(stepLog)
+					appVolumes, err = Inst().S.GetVolumes(ctx)
+
+					// Get the app volume details
+					log.FailOnError(err, "Failed to get volumes for app %s", ctx.App.Key)
+					dash.VerifyFatal(len(appVolumes) > 0, true, "App volumes exist?")
+					log.Infof("App volumes details: %v ", appVolumes)
+
+					// Store the volume details
+					for _, vol := range appVolumes {
+						apivol, err := Inst().V.InspectVolume(vol.ID)
+						if err != nil {
+							log.FailOnError(err, "failed to inspect volume %v", vol.ID)
+						}
+						volumrlidttr = append(volumrlidttr, apivol)
+					}
+
+					// Loop through the apps and check if the volumes are fastpath active before reboot
+					for _, appvolume := range appVolumes {
+						log.Infof("current volume : %v", appvolume.Name)
+						if strings.Contains(ctx.App.Key, fastpathAppName) {
+							err := ValidateFastpathVolume(ctx, opsapi.FastpathStatus_FASTPATH_ACTIVE)
+							log.FailOnError(err, "fastpath volume validation failed")
+						}
+					}
+				})
+			}
+		})
+		stepLog = " Step 4: Reboot the node wait for it to complete"
+		Step(stepLog, func() {
+			var err error
+			var volExists bool
+			log.Infof(" Before reboot check if the volumes are attached on the local node")
+			for _, volumePtr := range volumrlidttr {
+				volExists, err = Inst().V.IsVolumeAttachedOnNode(volumePtr, pxNode)
+				if !volExists || err != nil {
+					log.FailOnError(err, "Volume attached on local node validation failed")
```

Comment:
if err is nil, log.FailOnError will not fail

Reply:
Thanks for pointing that FailOnError already does  `if err != nil ` check. fixed it